src/i18n-strings/pagination.ts

```diff
@@ -2,5 +2,8 @@
 // SPDX-License-Identifier: MIT-0
 import { TableProps } from '@cloudscape-design/components';
 
-export const renderAriaLive: TableProps['renderAriaLive'] = ({ firstIndex, lastIndex, totalItemsCount }) =>
-  `Displaying items ${firstIndex} to ${lastIndex} of ${totalItemsCount}`;
+export const renderAriaLive: TableProps['renderAriaLive'] = ({ firstIndex, lastIndex, totalItemsCount }) => {
```

Comment:
Do we need to update any unit tests here (or possibly add any since I'm assuming before when `totalItemsCount` was undefined this behaviour caused a bug, so adding a unit test can be good)

Reply:
We can explicitly verify the live region is correct for those pages where the totalItemsCount is not provided, but going forward I think it is better to always defined the totals. Besides, the change will be implicitly covered by https://github.com/cloudscape-design/components/pull/2166